src/lib/types.js

```diff
@@ -81,6 +81,9 @@ export class Wei extends Immutable.Record({ val: ZERO }) {
     }
 }
 
+// TODO: get from gastracker api?
+export const GASPRICE_ESTIMATE = new Wei(23000000000);
```

Comment:
there's a getGasPrice in `store/accountActions`, it fetches gasprice on load and stores it in the state.